yaml-tests/src/main/java/com/apple/foundationdb/relational/yamltests/YamlTestExtension.java

```diff
@@ -100,6 +114,9 @@ public void afterAll(final ExtensionContext context) throws Exception {
         if (exception.isPresent()) {
             throw exception.get();
         }
+        for (ExternalServer server : servers) {
+            server.stop();
```

Comment:
I think we still want to make sure that it keeps stopping servers even if it fails to stop one.
Probably something similar to the above, or by making `stop` swallow and log exceptions.
We probably also want to stop the servers even if an `afterAll` throws an exception.

Reply:
The `destroy` method does not throw exceptions, but better to be safe. Done.